aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1400,7 +1401,18 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
                     deleteInlineStyleFromTheBeginning()
                 }
             }
-            else -> return super.onTextContextMenuItem(id)
+            // Fix for crash when pasting text on Samsung Devices running Android 8.
+            // Ref: https://github.com/wordpress-mobile/WordPress-Android/issues/8827
+            16908904,
```

Comment:
This value on my device : Samsung SM-T585 Android 8.1 is different -> 16908872.

Reply:
Interesting.... `16908874` --> was from Android 8, and `16908904` from Android 9. I just added it since don't know if 8.X will get the latest version of their clipboard.

I guess i can add your other value to the list, but it's like going per-case. Anyway don't have better idea, since the src code of their clipboard isn't available.

Reply:
@daniloercoli let me try something, and if doesn't work I will give the Approve so that we can move on.